fix(tutorials): tutorial review MTA-1689 part 1 (#428)
scaleway/docs-content: tutorials/encrypt-volumes/index.mdx

---
meta:

>   title: Encrypting Volumes for zensitive eata

Author: ```suggestion
  title: Encrypting Volumes for sensitive data
```

>   description: This tutorial will teach you how to create and mount an encrypted volume for sensitive data.

Reviewer: ```suggestion
  description: This tutorial shows you how to create and mount an encrypted volume for sensitive data.
```

content:

>   h1: Encrypting Volumes for sensitive data

Author: ```suggestion
  h1: Encrypting volumes for sensitive data
```

>   paragraph: This tutorial will teach you how to create and mount an encrypted volume for sensitive data.

Reviewer: ```suggestion
  paragraph: This tutorial shows you how to create and mount an encrypted volume for sensitive data.
```

tags: compute encrypting volumes sensible-data
categories:
  - compute
  - storage
  - instances
dates:
  validation: 2022-02-22
  posted: 2019-02-04
---

> ## Encrypted Volumes Overview

Reviewer 2: ```suggestion
## Encrypted volumes overview
```
